model/src/main/proto/survey.proto

```diff
@@ -122,17 +125,17 @@ enum UserTypeAnswer {
   // User type unknown.
   USER_TYPE_UNSPECIFIED = 0;
 
-  // Corresponds to an unspecified or unknown type of user.
-  OTHER = 1;
+  // Corresponds to a user who is a learner.
+  LEARNER = 1;
 
-  // Corresponds to a user who is a teacher.
+  // Corresponds to a user who is a parent.
```

Comment:
This comment and the next one are incorrectly switched.

Reply:
Corrected